lcm/lcm_coretypes.h

```diff
@@ -7,6 +7,11 @@
 #include <stdlib.h>
 
 #ifdef __cplusplus
+#if defined(__clang__)
+// https://bugs.llvm.org/show_bug.cgi?id=37350
```

Comment:
Okay, looking closer... this is disingenuous; clang isn't doing anything "wrong". A better comment would be "Suppress warnings about C-style casts, since this code needs to build in both C and C++ modes".

Reply:
Done.